int_experian/int_experian_sgjc/cartridge/templates/default/edq/verificationEngine.isml

```diff
@@ -3,12 +3,14 @@
 TASK:101728 Change Validate Button
 </iscomment>
 <isif condition="${URLUtils.continueURL().toString().search('Address') != -1}">
-	<button class="apply-button" type="button" id="form-submit"><span>${Resource.msg('global.apply','locale',null)}</span></button>
+	<button type="button" id="form-submit"><span>${Resource.msg('global.apply','locale',null)}</span></button>
 </isif>
 <isif condition="${URLUtils.continueURL().toString().search('COShipping') != -1}">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
```

Comment:
Moved those libraries here because they were added automatically in the Address page; but no t in the Billing and Shipping page; in address page generated double call for the library.